sql/wsrep_server_service.cc

```diff
@@ -146,6 +146,7 @@ void Wsrep_server_service::release_high_priority_service(wsrep::high_priority_se
 void Wsrep_server_service::background_rollback(wsrep::client_state& client_state)
 {
   Wsrep_client_state& cs= static_cast<Wsrep_client_state&>(client_state);
```

Comment:
Note: No need for special handling for `KILL` command here, `KILL` does not go through `wsrep-lib` in this PR.